### PR TITLE
Handle W3C API new group URLs

### PR DESCRIPTION
**Pull request by deniak**
The W3C API now lists the group links based on their type/shortname, e.g. https://api.w3.org/groups/wg/payments.
While the W3C API still support the URLs with the group ID (https://api.w3.org/groups/83744), ash-nazg needs to be updated to check the new URLs.

The W3C API has been temporarily reverted so it doesn't break ash-nazg. Once that PR is merged, the API will have to be updated again.

### CODE REVIEW

File: pr-check.js [hunk 1]
```diff
@@ -171,7 +171,25 @@ function prChecker(config, argLog, argStore, GH, mailer) {
             pr.contribStatus[username] = "undetermined affiliation";
             return "undetermined affiliation";
           }
-          let result = await w3ciprcheck(w3c, user.w3capi, user.displayName, repoGroups, store);
+
+          let groups = [];
+          const types = {
```
Comment (dontcallmedom) [inline pr-check.js:176]: nit: this could be defined as a module-level object since it is essentially immutable

**Review by dontcallmedom — APPROVED** [review]

**Comment by dontcallmedom** [comment]
I guess the release needs to be coordinated with the re-release of the W3C API upgrade?

File: pr-check.js (hunk 1 above)
Comment (deniak) [reply pr-check.js]: Thanks! PR updated.

**Comment by deniak** [comment]
> I guess the release needs to be coordinated with the re-release of the W3C API upgrade?

Yes, I'll update the API when I deploy this PR.
Thanks for the review.

**Comment by deniak** [comment]
The PR has now been deployed and the W3C API updated.

**Pull request merged**